_data/changelogs/docs-settings.yml

```diff
@@ -2,6 +2,17 @@ title: Settings
 type: utility
 changelogURL:
 items:
+  - date: 2023-05-12
+    summary: Updated sass map instructions to reflect new pattern.
+    summaryAdditional: Users are now instructed to define only the variable they wish to change instead of having to define the entire map
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance: true
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles:
```

Comment:
```suggestion
    affectsStyles: true
```

Reply:
Resolved in [e6b26d8](https://github.com/uswds/uswds-site/pull/2077/commits/e6b26d84f667f9e2ab669bd51dbb70d345bc6b64)